python/lsst/dax/apdb/cli/apdb_cli.py

```diff
@@ -78,6 +80,28 @@ def _create_cassandra_subcommand(subparsers: argparse._SubParsersAction) -> None
     parser.set_defaults(method=scripts.create_cassandra)
 
 
+def _list_cassandra_subcommand(subparsers: argparse._SubParsersAction) -> None:
+    parser = subparsers.add_parser("list-cassandra", help="List APDB instances in Cassandra cluster.")
+    parser.add_argument("host", help="One of the host names for Cassandra cluster.")
+    parser.set_defaults(method=scripts.list_cassandra)
+
+
+def _delete_cassandra_subcommand(subparsers: argparse._SubParsersAction) -> None:
+    parser = subparsers.add_parser("delete-cassandra", help="Delete APDB instance from Cassandra cluster.")
+    parser.add_argument("host", help="One of the host names for Cassandra cluster.")
+    parser.add_argument(
+        "keyspace", help="Cassandra keyspace name for APDB tables, will be created if does not exist."
```

Comment:
Is it true that the keyspace will be created if it doesn't exist? I would rather see the delete function fail in that case.

Reply:
My fault, forgot to update it after copy/paste. It is indeed an error if keyspace does not exist, an exception will be raised in that case.